tests/assembly/targets/s390x_unknown_linux_gnu.rs

```diff
@@ -0,0 +1,17 @@
+// assembly-output: emit-asm
+// compile-flags: --target s390x-unknown-linux-gnu
+// needs-llvm-components: s390x
```

Comment:
@cuviper are you [saying](https://github.com/rust-lang/rust/pull/116030#discussion_r1333772262) this should say `systemz`?

Reply:
Yep, that's what the component is named in LLVM.